config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java

```diff
@@ -31,22 +31,29 @@
     private static final String REGISTRY_CONF_SUFFIX = ".conf";
     private static final String ENV_SYSTEM_KEY = "SEATA_ENV";
     private static final String ENV_PROPERTY_KEY = "seataEnv";
-    /**
-     * the name of env
-     */
-    private static String envValue;
+
+    private static final String SYSTEM_PROPERTY_SEATA_CONFIG_NAME = "seata.config.name";
```

Comment:
Can we add this to file.conf as a configuration item?

Reply:
I wanted to use a custom name of registry.conf, so added a system property `seata.config.name` here. I understand your meaning, but file.conf is configured in registry.conf. I think it's not possible to do this, is my understanding right?

Reply:
Sure.Your explanation makes sense.